packages/devtools_app/lib/src/screens/vm_developer/object_store.dart

```diff
@@ -0,0 +1,92 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+import 'package:vm_service/vm_service.dart';
+
+import '../../primitives/utils.dart';
+import '../../shared/common_widgets.dart';
+import '../../shared/table.dart';
+import '../../shared/table_data.dart';
+import 'object_store_controller.dart';
+import 'vm_developer_common_widgets.dart';
+import 'vm_service_private_extensions.dart';
+
+class _Entry extends ColumnData<ObjectStoreEntry> {
+  _Entry() : super.wide('Entry');
+
+  @override
+  bool get includeHeader => true;
+
+  @override
+  String getValue(ObjectStoreEntry dataObject) {
+    return dataObject.key;
+  }
+}
+
+class _Object extends ColumnData<ObjectStoreEntry> implements ColumnRenderer {
```

Comment:
nit: name _EntryColumn and _ObjectColumn to make it clear that these are column classes and not data classes